src/tour/TourManager.ts

```diff
@@ -397,6 +413,9 @@ export class TourManager {
 
 
   private setUp(tour: Tour, context: any = {}) {
+    window.addEventListener('resize', this.resizeListener, {
```

Comment:
Instead of `window` we could use `this.backdrop.ownerDocument.defaultView`, such that this functionality would also work in external windows for example.

Reply:
Good idea! I changed it.